test/topology/util.py

```diff
@@ -265,7 +265,7 @@ async def start_writes_to_cdc_table(cql: Session, concurrency: int = 3):
     await cql.run_async(f"CREATE TABLE {ks_name}.tbl (pk int PRIMARY KEY, v int) WITH cdc = {{'enabled':true}}")
 
     stmt = cql.prepare(f"INSERT INTO {ks_name}.tbl (pk, v) VALUES (?, 0)")
-    stmt.consistency_level = ConsistencyLevel.ONE
+    stmt.consistency_level = ConsistencyLevel.TWO
```

Comment:
nit: QUORUM would be preferred (more general, would work with 5 node scenario etc.)

but this change is not worth restarting CI

Reply:
I forgot about this nit. I'll mention that QUORUM is an option in a new issue.